src/locale.js

```diff
@@ -76,25 +76,48 @@ export default class Locale {
 		this.contentLanguageDirection = getLanguageDirection( this.contentLanguage );
 
 		/**
-		 * Translates the given string to the {@link #uiLanguage}. This method is also available in
-		 * {@link module:core/editor/editor~Editor#t} and {@link module:ui/view~View#t}.
+		 * Translates the given string to the {@link #uiLanguage}.
 		 *
-		 * The strings may contain placeholders (`%<index>`) for values which are passed as the second argument.
-		 * `<index>` is the index in the `values` array.
+		 * @method #t
+		 * @param {String} msgid The string to translate.
+		 * @returns {Message}
```

Comment:
Does it return `Message` or `String`?

Reply:
I think it can return `Message`, if it is stringify-able easily. But then we need to change `t()` function definition.